src/utils/client_test.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"testing"
+)
+
+func Test_configHarborWithSetting(t *testing.T) {
```

Comment:
This test case not follow Golang test name convention.

Reply:
update